integration-test/src/commonTest/kotlin/sample/JsonTest.kt

```diff
@@ -212,22 +209,6 @@ class JsonTest {
         )
     }
 
-    @Test
-    fun testZoo() {
```

Comment:
Have you moved it somewhere or just deleted?

Reply:
BasicTypesSerializationTest

Reply:
`BasicTypesSerializationTest` is available in `runtime` module, but not in the `integration-test`